src/main/java/lotto/utils/StringUtils.java

```diff
@@ -0,0 +1,44 @@
+package lotto.utils;
+
+public final class StringUtils {
+
+    private StringUtils() { }
+
+    public static boolean isNullOrEmpty(final String text) {
+
+        if ( text == null ) {
+            return true;
+        }
+
+        if ( text.isEmpty() ) {
+            return true;
+        }
+
+        if ( text.trim().isEmpty() ) {
+            return true;
+        }
+
+        return false;
```

Comment:
아래의 코드처럼 한 줄로 줄일 수 있어요.

```suggestion
        return Objects.isNull(text) || text.trim().isEmpty();
```